pkg/platform/model/vcs.go

```diff
@@ -888,23 +888,30 @@ func GetRevertCommit(from, to strfmt.UUID) (*mono_models.Commit, error) {
 	return res.Payload, nil
 }
 
-func RevertCommitWithinHistory(from, to strfmt.UUID) (*mono_models.Commit, error) {
-	ok, err := CommitWithinCommitHistory(from, to)
+func RevertCommitWithinHistory(from, to, latest strfmt.UUID) (*mono_models.Commit, error) {
+	ok, err := CommitWithinCommitHistory(latest, from)
```

Comment:
Maybe we should also check if "to" is also within the commit history.

Reply:
If the 'to' commit is the initial commit, then the platform will return an error because it cannot fetch any prior history, and thus cannot verify if the commit is within its history.